command/server/init.go

```diff
@@ -71,6 +78,18 @@ func (p *serverParams) initBlockTime() error {
 	return nil
 }
 
+func (p *serverParams) initIBFTBaseTimeout() error {
+	if p.rawConfig.IBFTBaseTimeout < 1 {
+		return errInvalidIBFTBaseTimeout
+	}
+
+	if p.rawConfig.IBFTBaseTimeout <= p.rawConfig.BlockTime {
+		return errWrongIBFTBaseTimeout
+	}
```

Comment:
What do you think about setting the `IBFTBaseTimeout` to be some kind of multiple of `BlockTime`?

For example, the default `BaseTimeout` is `10s`, but the block time is `15s` -> you could set the `BaseTimeout` to be `20s`

cc @dbrajovic

Reply:
This is of course if you don't explicitly set `BaseTimeout` with the flag

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/582/commits/219c6c93359a3088eb0ed95c0fe2006cc5ea708c